src/bank/accounts/Account.java

```diff
@@ -0,0 +1,49 @@
+package bank.accounts;
+
+/**
+ * The abstract Account class serves as the base class for various types of bank accounts.
+ * It defines common properties and methods that are shared among all account types.
+ * @author Kobby Asante-Ansong
+ */
+public abstract class Account {
+
+    /** The balance of the account. */
+    public double balance;
+
+    /** The successor account, used in case of insufficient funds for CoR. */
+    public Account successor;
+
+    /** The name associated with the account. */
+    public String name;
```

Comment:
Attributes should be private or protected unless they are constants then they would also be final.